types/index.d.ts

```diff
@@ -3503,7 +3503,7 @@ export interface ChartTypeRegistry {
   bar: {
     chartOptions: BarControllerChartOptions;
     datasetOptions: BarControllerDatasetOptions;
-    defaultDataPoint: number | null;
+    defaultDataPoint: number | number[] | null;
```

Comment:
Since it always uses 2 entries this is a safer and more descriptive option
```suggestion
    defaultDataPoint: number | [number, number] | null;
```